Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocList.css

```diff
@@ -107,7 +107,7 @@ div.bocListMenuBlock
 {
   display: block;
   float: right;
-  width: 10em;
+  width: var(--boclist-menublock-minimum-width, 10em);
```

Comment:
I'm not investigating the previous comments, so I'll just ask: did I want to support min/max for NovaGray, too, or just use the minvalue as an absoluete?

Reply:
Either implement Grid layout for novagray or use the minvalue as absolute width. I chose the second approach

Reply:
oh, wait, I'm probably needing better glasses. this is `var(value, default)` not `minmax(value, othervalue)`